packages/toolchain/dasel/build.yaml

```diff
@@ -0,0 +1,20 @@
+requires:
+  - name: "golang"
+    category: "build"
+    version: ">=0"
+env:
+  - PATH=$PATH:/usr/local/go/bin
```

Comment:
```suggestion
  {{ template "golang_env" }}
{{if .Values.fips}}
- CGO_ENABLED=1
{{ else }}
- CGO_ENABLED=0
- LDFLAGS="-s -w"
{{ end }}
```

Reply:
This would also remove the next 2 lines, but github doesnt allow me to add the suggestion for multilines :/

Reply:
gotcha, fixing it from cli